server-base/src/main/java/org/apache/kylin/rest/controller/CubeController.java

```diff
@@ -855,6 +863,38 @@ public CubeRequest updateCubeDesc(@RequestBody CubeRequest cubeRequest) throws J
 
     }
 
+    /**
+     * resubmit streaming segment
+     *
+     * @throws IOException
+     */
+    @RequestMapping(value = "/{cubeName}/segs/{segmentName}/resubmit", method = {RequestMethod.PUT}, produces = {"application/json"})
```

Comment:
How about adding this api as an action item for a streaming cube?

Reply:
May I need to add a 'resubmit segment' action item to the streaming cube page?